packages/pwa-kit-react-sdk/src/ssr/universal/components/with-react-query/index.js

```diff
@@ -17,8 +17,7 @@ const STATE_KEY = '__reactQuery'
  *
  * @param {React.ReactElement} Wrapped The component to be wrapped
  * @param {Object} options
- * @param {Object} options.queryClientConfig The react query client configuration object to be used. By
- * default the `retry` option will be set to false to ensure performant server rendering.
```

Comment:
we used to have a default options in `useReactQuery` implementation, but we removed it, and forgot to remove the jsdoc comment.